precompiles/avs/query.go

```diff
@@ -102,7 +115,111 @@ func (p Precompile) GetOperatorOptedUSDValue(
 	}
 	amount, err := p.avsKeeper.GetOperatorKeeper().GetOperatorOptedUSDValue(ctx, avsAddr.String(), operatorAddr)
 	if err != nil {
+		if errors.Is(err, avstype.ErrNoKeyInTheStore) {
+			return method.Outputs.Pack(common.Big0)
+		}
 		return nil, err
 	}
 	return method.Outputs.Pack(amount.ActiveUSDValue.BigInt())
 }
+
+func (p Precompile) GetAVSInfo(
+	ctx sdk.Context,
+	_ *vm.Contract,
+	method *abi.Method,
+	args []interface{},
+) ([]byte, error) {
+	if len(args) != len(p.ABI.Methods[MethodGetAVSInfo].Inputs) {
+		return nil, fmt.Errorf(cmn.ErrInvalidNumberOfArgs, len(p.ABI.Methods[MethodGetAVSInfo].Inputs), len(args))
+	}
+	addr, ok := args[0].(common.Address)
+	if !ok {
+		return nil, fmt.Errorf(exocmn.ErrContractInputParaOrType, 0, "common.Address", addr)
+	}
+
+	avs, err := p.avsKeeper.QueryAVSInfo(ctx, &avstype.QueryAVSInfoReq{AVSAddress: addr.String()})
```

Comment:
Why do you use the `QueryAVSInfo` interface instead of `GetAVSInfo` directly?

Reply:
fixed